tests-integrations/flat-config/eslint.config.js

```diff
@@ -0,0 +1,15 @@
+import intlifySvelte from '@intlify/eslint-plugin-svelte'
+import svelteParser from 'svelte-eslint-parser'
+
+export default [
+  ...intlifySvelte.configs['flat/recommended'],
+  {
+    files: ['**/*.svelte', '*.svelte'],
+    languageOptions: {
+      parser: svelteParser
+    },
+    rules: {
+      '@intlify/svelte/no-raw-text': 'error'
```

Comment:
We probably need to remove this. The rule needs to be flagged with a warning, otherwise execSync will throw an error.
Or we need to gather the reports in another way that doesn't use execSync.

Reply:
Okay, so I'll update the code to work with the first option

Reply:
It worked locally but failed on the CI

Reply:
Just tried the second options. Also fails on CI

Reply:
> It worked locally but failed on the CI

I think it's failing with yarn install, it seems like it needs to install ignoring dependency version compatibility.

Reply:
Ah, it works! Thanks a lot @ota-meshi